### ISSUE TITLE
Avocode can not be installed on Ubuntu 19.10

### ISSUE THREAD

**Issue by lubomirblazekcz** [avocode/avocode#2314]
* **Version**: the one listed here https://help.avocode.com/en/articles/513807-installing-avocode-on-linux

* **Web/Desktop**: Desktop

* **Platform/Browser**: Ubuntu 19.10

 

Avocode can not be installed on Ubuntu 19.10, because it depends on gir1.2-gnomekeyring-1.0 which is depricated and not available on ubuntu 19.10 packages. This package is still downloadable on 19.04, but manual install did not work for me.

```Reading package lists... Done
Building dependency tree       
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 avocode : Depends: gir1.2-gnomekeyring-1.0 but it is not installable
E: Unable to correct problems, you have held broken packages.
```

Avocode itself works for me just fine when downloaded manually and launched via attached process in terminal. But installed version would be more preferable in the future :)

**Comment by jakubzitny** [comment]
Thanks for the report @evromalarkey, we'll have a look at it 🙏 🐧

**Comment by jakubzitny** [comment]
We have a fix ready, it will be included in the next release of Avocode 🥑

**Comment by lubomirblazekcz** [comment]
Thanks a lot 👍

**Comment by jakubzitny** [comment]
We just released `4.1.0` where this is fixed. Please, let us know if there are any more problems 😉

**Comment by lubomirblazekcz** [comment]
Works great, thanks!
